### PR TITLE
Fix formatter not being used.

### PR DESCRIPTION
**Pull request by marek**
This will take the format specifier defined in the configuration and treat is as json rather than a string. If possible it will merge it with the json message sent to logzio.

### CODE REVIEW

**Comment by roiravhon** [comment]
Thanks for this @marek!
Ill test the change and merge.
Can take a couple of days

File: logzio/handler.py [hunk 1]
```diff
@@ -102,14 +102,16 @@ def add_to_buffer(self, message):
             self.logs_counter_condition.release()
 
     def handle_exceptions(self, message):
-        if message.exc_info:
-            return '\n'.join(traceback.format_exception(*message.exc_info))
-        else:
-            return message.getMessage()
+        message = '\n'.join(traceback.format_exception(*message.exc_info))
```
Comment (robertaistleitner) [inline logzio/handler.py]: there's an obvious problem here.. this method doesn't return the generated message..

**Comment by marek** [comment]
@robertaistleitner I addressed the issues with a few more commits.

**Comment by marek** [comment]
bump.  Wondering if there is anything else I can address or your thoughts?

**Comment by robertaistleitner** [comment]
Sorry but I'm not the project owner, I was just passing by here because I checked out logzio support for different languages.. :wink:

**Comment by roiravhon** [comment]
Hey @marek, so sorry for the delay, i was swamped the last few days so didn't got the time to actually test this.
The PR looks fine, and thanks @robertaistleitner for the comment!

I'll update soon when merged and pushed to pypi

**Comment by roiravhon** [comment]
Thanks a lot @marek, tested and pushed to pypi, should be available to pip with version 1.0.4

Appreciate your contribution!

**Pull request merged**